yklua/bench.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/sh
+
+set -eu
+
+lua=$1
+
+if [ ! -f "${lua}" ]; then
+    printf "Cannot find %s executable!\n" "${lua}"
```

Comment:
Generally prefer `echo "Executable '$lua' not found" > /dev/stderr` or similar.

Reply:
done :point_right: 1d7ab91c3aff41bc0424b1495be15d57e886e223